setup/install-surelog.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/sh
 
+set -e
+
 # Get directory of script
 src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
 
-sudo apt-get install -y build-essential cmake git pkg-config tclsh swig uuid-dev libgoogle-perftools-dev python3 python3-dev
-sudo apt-get install -y default-jre
-
-pip3 install orderedmultidict
+# These dependencies are up-to-date with instructions from the INSTALL.md from the commit we are pinned to below
+sudo apt-get install -y build-essential cmake git pkg-config tclsh swig uuid-dev libgoogle-perftools-dev python3 python3-orderedmultidict python3-psutil python3-dev default-jre lcov
```

Comment:
I'm not sure about installing PyPi packages using the Ubuntu package repositories - is there a reason that we shouldn't be consistent about installing Python package dependencies through Python's package manager?

Reply:
This makes this script consistent with the install instructions

Reply:
Okay, makes sense - and I guess that we wouldn't want the Surelog installation to depend on a Python virtual environment, anyways.

Reply:
We need these from: https://github.com/chipsalliance/Surelog/blob/master/INSTALL.md

Reply:
Unfortunately, without the Ubuntu packages, Surelog will not install. I've tried with and without venvs and Surelog is still looking for the `orderedmultidict` package from the official python package area.